examples/nextjs-with-typescript/pages/MediaChromeReactSimple.tsx

```diff
@@ -0,0 +1,92 @@
+import type { NextPage } from 'next';
+import Head from 'next/head';
+import 'media-chrome';
+import styles from '../styles/Home.module.css';
+// @ts-ignore
+import {
+  Provider as MediaChromeProvider,
+  useMediaUIDispatch as useDispatch,
+  useMediaUISelector as useSelector,
+} from '../components/media-chrome-react/stateMgmt';
+
+const PlayButton = () => {
+  const dispatch = useDispatch();
+  // @ts-ignore
+  const mediaPaused = useSelector((state) => state.mediaPaused);
+  return (
+    <button
+      style={{ cursor: 'pointer' }}
+      onClick={() => {
+        const type = mediaPaused ? 'mediaplayrequest' : 'mediapauserequest';
+        dispatch({ type });
+      }}
+    >
+      {mediaPaused ? 'Play' : 'Pause'} React Only!
+    </button>
+  );
+};
+
+const Video = () => {
+  const dispatch = useDispatch();
+  return (
+    <video
+      ref={(videoEl) =>
+        dispatch({ type: 'mediaelementchangerequest', detail: videoEl })
+      }
+      slot="media"
+      src="https://stream.mux.com/A3VXy02VoUinw01pwyomEO3bHnG4P32xzV7u1j1FSzjNg/high.mp4"
+      muted
+      crossOrigin=""
+      playsInline
+    ></video>
+  );
+};
+
+const MediaPlayButton = () => {
+  const dispatch = useDispatch();
```

Comment:
Not sure if this would need to be solved in this iteration/v1.0 requirements, but we may want to (in some manner or another) account for "standalone usage" with the react wrapper components if we go in this direction. Food for thought.